quickstarts/quickstart-template.md

```diff
@@ -0,0 +1,88 @@
+ 
+
+# Title
+
+:information_source: Before using this template, read the accompanying [quickstart template guide](about-quickstarts.md).
+
+**Document Version:** {MAJOR.MINOR.PATCH} 
+
+**Last Updated:** {"Month DD, YYYY" or "Month YYYY"}   
+
+**Application Version:** 
+
+- {Application 1}: {MAJOR.MINOR.PATCH}
+- {Application 2}: {MAJOR.MINOR.PATCH} or later
+- ...
```

Comment:
**Suggestion (blocking):** I recommend removing the metadata until the template working group leads can reach a consensus about it.

Reply:
Fixed